onnx/defs/printer.cc

```diff
@@ -319,7 +319,10 @@ void ProtoPrinter::print(const AttrList& attrlist) {
 void ProtoPrinter::print(const NodeProto& node) {
   output_ << std::setw(indent_level) << ' ';
   printSet("", ", ", "", node.output());
-  output_ << " = " << node.op_type();
+  output_ << " = ";
+  if (node.domain() != "")
+    output_ << node.domain() << ".";
```

Comment:
"ai.onnx" and "" both mean the default domain. Should ops with domain "ai.onnx" have consistent behavior with those with "" domain?

Reply:
yes, that's right. (Does it have any implication to this PR?)

Reply:
> Does it have any implication to this PR?

Yes, maybe the line 323 should be changed to `if (node.domain() != "" && node.domain() != "ai.onnx")`

Reply:
I think it is better in the current form. Both forms are legal and mean the same thing. The parser/printer are just convenience utilities to convert the proto format to/from a readable textual form. It is useful to preserve the distinction between the two different, but equivalent, forms in the parser/printer.